### PR TITLE
Documentation: Console

### PR DESCRIPTION
**Pull request by aspeddro**

### CODE REVIEW

File: src/Core__Console.res [hunk 1]
```diff
@@ -24,5 +24,5 @@
 
 @val external trace: unit => unit = "console.trace"
 
-@val external timeStart: string => unit = "console.timeStart"
+@val external time: string => unit = "console.time"
```
Comment (aspeddro) [inline src/Core__Console.res:27]: `console.timeStart` don't exits. See [Console API](https://developer.mozilla.org/en-US/docs/Web/API/console)

**Comment by zth** [comment]
This is great work @aspeddro ! 👏

Reviewing soon.

**Review by zth — COMMENTED** [review]
This is looking great overall! However, since descriptions are copied from MDN, we need to figure out what to do about licenses. MDN descriptions are licensed as CC-BY-SA, so we need to give attribution. Question is whether we need to give attribution right next to every text, or if it's enough to give attribution somewhere central in the repo. Need to read up on that.

Another alternative is rewriting in our own words, taking inspiration from MDN and other sources, but not copying.

**Comment by aspeddro** [comment]
Ok, I'll rewrite.

**Pull request merged**